content/docs/command-reference/plots/index.md

```diff
@@ -111,10 +111,10 @@ DVC has the following built-in plot templates:
 ### Custom templates
 
 Plot template files are
-[Vega specification](https://vega.github.io/vega/docs/specification/) files that
-use predefined DVC anchors as placeholders for DVC to inject the plot values.
-You can create a custom template from scratch, or modify an existing one from
-`.dvc/plots/`.
+[Vega specification](https://vega.github.io/vega/docs/specification/) JSON or
+HTML files that use predefined DVC anchors as placeholders for DVC to inject the
+plot values. You can create a custom template from scratch, or modify an
+existing one from `.dvc/plots/`.
```

Comment:
Ah but wait. This doesn't seem correct. The change should be mainly in the previous section. This one also needs updates I suppose, since "Plot template files are Vega spec files" is incorrect. But the changes here are not more correct. Please re-read the discussion in https://github.com/iterative/dvc/pull/3830#pullrequestreview-415640978 and try again or feel free to ask for help. The Vega stuff is kind of advanced/specific.

Closing for now.